doc/tutorials/core/adding_images/adding_images.markdown

```diff
@@ -1,13 +1,16 @@
 Adding (blending) two images using OpenCV {#tutorial_adding_images}
 =========================================
 
+@prev_tutorial{tutorial_mat_operations}
+@next_tutorial{tutorial_basic_linear_transform}
+
 Goal
 ----
 
 In this tutorial you will learn:
 
 -   what is *linear blending* and why it is useful;
--   how to add two images using @ref cv::addWeighted
+-   how to add two images using **addWeighted()**
```

Comment:
I would suggest to add another method
```cpp
// addWeighted( src1, alpha, src2, beta, 0.0, dst);
dst = alpha * src1 + beta * src2 + 0.0;
```
which is more intuitive and user friendly, as we do not need to remember which function to call.